### ISSUE TITLE
mToolkit Eclipse Plugin does not work under Eclipse Neon

### ISSUE THREAD

**Issue by mcarrer** [eclipse/kura#496]
The mToolkit Eclipse Plugin is no longer downloadable from the original location and it no longer works under Eclipse Neon.

We have put together the following remediation plan:

1. We will update the Kura development guides documenting the limitation of mToolkit for Eclipse Neon and Mars.

2. In the same documentation pages, we will add a note on how to create a deployment package outside of Eclipse.
We are evaluating a couple of options on this and we will try to keep it as simple as possible

3. For those using older versions of Eclipse, we will provide the alternative download URL:
http://mtoolkit.stage.tigris.org/source/browse/mtoolkit/

4. Finally, we will reach out to the wider Eclipse community to see if anybody is willing to update the mToolkit project.

**Comment by pgufAtIBM** [comment]
today I saw a mail from Merlino Pierantonio:
Hi guys, 

in the last months the mToolkit site appeared to be down or intermittently up. 
Moreover, the mToolkit plugin for Eclipse is not updated and it does not support the new Eclipse Neon.

So, we created a couple of new Update Sites in order to make the mToolkit plugin available to all. 
The installation procedure is the same as the old site, but the new address is:

http://mtoolkit-mars.s3-website-us-east-1.amazonaws.com for Eclipse Mars and previous versions
http://mtoolkit-neon.s3-website-us-east-1.amazonaws.com for Eclipse Neon

Best,
Pier

I tested in on Neon -> it works there
